src/main/scala/com/gd/twitteranalytics/util/TweetsConstant.scala

```diff
@@ -0,0 +1,11 @@
+package com.gd.twitteranalytics.util
+
+object TweetsConstant {
+
+  val PARTITION_COLUMN = "event_date"
+  val SAVEPATH = "twitter-analytics/dataLake/eng-tweets"
+  val TABLENAME = "engtweets"
```

Comment:
could you clarify, what is this for?

Reply:
saveAsTable needs a TableName
def saveAsTable(tableName: String): Unit 
though in the output, I don't see it using this table name.

<img width="354" alt="screen shot 2018-09-24 at 4 03 49 pm" src="https://user-images.githubusercontent.com/42447829/45983925-974b3480-c013-11e8-893a-4a77ac85fd79.png">

Reply:
I didn't find a usage of the constant because `TweetsDataSave` was not present.

Reply:
> though in the output, I don't see it using this table name.

please figure this out -  if the table name is required and why.

Reply:
Have update the code to remove option and saveAsTable to be replaced by save(path) only. It doesn't need a Table name now.
If we give only table name, it creates a table directly in warehouse.

Reply:
don't see a commit with the change. the latest one was published on 24/09.

Reply:
A couple of review comments were pending to implement. So code was not pushed.

Reply:
@shSachdeva 
Comments like "Fixed" / "Done" / "Updated ... " (notifying that comment was resolved) should be posted after/together with a corresponding commit. So, reviewer can view the actual change / see evidence of comment resolution.